### PR TITLE
Frontend test

### PR DESCRIPTION
**Pull request by mattdean1**
Probably the best entrypoint to review this is to look at the readme :) 

Would also be amazing if you could clone and test out the install / run instructions on a fresh machine!

### CODE REVIEW

File: frontend/README.generated.md [hunk 1]
```diff
@@ -0,0 +1,819 @@
...
+
+You can use that file while generating a new project to reuse all those variables.
+
+For example, run:
+
+```console
+$ cookiecutter --config-file ./cookiecutter-config-file.yml --output-dir ../project-copy https://github.com/tiangolo/full-stack-fastapi-postgresql
+```
+
+That will use the file `cookiecutter-config-file.yml` in the current directory (in this project) to generate a new project inside a sibling directory `project-copy`.
```
Comment (aoodham) [inline frontend/README.generated.md:819]: an 819 line readme is a _little_ bit overwhelming for a tech test?

File: frontend/README.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
+# Net Purpose frontend tech test 🎨
+
+## What is the task?
+
+Your task is to create a frontend "admin dashboard" for some fictional portfolio holdings. Portfolio details will be delivered via the API (see below). We're expecting you to spend a few hours on this.
```
Comment (aoodham) [inline frontend/README.md:5]: This is quite a big ask

File: frontend/README.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+- [Yarn](https://yarnpkg.com/getting-started/install)
+
+### Start the app
+
+```
+# in the backend directory
+docker-compose up -d
+
+# in the frontend directory
+yarn && yarn start
```
Comment (aoodham) [inline frontend/README.md:105]: is yarn required? I feel like lowest common denominator is probably better

**Review by aoodham — COMMENTED** [review]
I worry that there's lots of baggage in this which makes it quite intimidating. e.g. the celery stuff, the emails etc.

I also think we may need to be a bit more prescriptive in what we want them to do with the admin dashboard. Otherwise we're testing people's creativity in the abstract, which isn't perhaps that fair.

File: frontend/README.md (hunk 2 above)
Comment (aoodham) [reply frontend/README.md:5]: The few hours thing that is

File: frontend/README.md (hunk 3 above)
Comment (mattdean1) [reply frontend/README.md:105]: that's the default with CRA these days

**Pull request merged**